### DISCUSSION TITLE
Welcome to aiogram Discussions!

### DISCUSSION THREAD

**Discussion by JrooTJunior** [aiogram/aiogram#476] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lamroy95** [comment]
**Aiogram cool**

**Reply by Arwichok** [reply]
When 3.0?)

**Reply by evgfilim1** [reply]
@Arwichok in Autumn in March

**Comment by OCCASS** [comment]
Hello, I'm a young programmer. And I want to introduce you the first library for testing bots written on aiogram3. Links: [GitHub](https://github.com/OCCCAS/aiogram_tests), [PyPi](https://pypi.org/project/aiogram-tests/). 
I will gladly accept the criticism and will spread it (telegram: @hui_s_gor)

If you liked the library, you can put ⭐️)

**Reply by makhmud-dev** [reply]
super bro

**Comment by MisanoGo** [comment]
Hello, 
Since Docs is not usefull and must work with trial and error
